pkgs/by-name/te/testlib/package.nix

```
@@ -0,0 +1,34 @@
+{
+  fetchFromGitHub,
+  stdenv,
+  lib,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "testlib";
+  version = "0.9.41";
+
+  src = fetchFromGitHub {
+    owner = "MikeMirzayanov";
+    repo = "testlib";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-AttzDYLDlpfL3Zvds6yBR/m6W/3UZKR+1LVylqOTQcw=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    install -Dt $out/include/testlib testlib.h
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "C++ implementation of testlib";
```

Actually the GitHub project's description is much more informative.
```suggestion
    description = " C++ library to develop competitive programming problems";
```